pyproject.toml

```diff
@@ -24,7 +25,7 @@ dependencies = [
   'pooch',
   'scooby>=0.5.1',
   'typing-extensions',
-  'vtk<9.4.0',
+  'vtk',
```

Comment:
The version pinning for other version should be handled in a separate PR.  Feel free to leave here temporarily for testing, but it should be removed before merging.